### ISSUE TITLE
Marker does not use a unique id

### ISSUE THREAD

**Issue by richardmward** [bpmn-io/bpmn-js#706]
Hi,

I am using bpmn-js in a tabbed layout, meaning there are multiple bpmn diagrams in a single HTML document. Mostly this works absolutely fine, except that the arrows on sequence flows only work in the first diagram in the page. 

I have done some digging and I think the cause is the way that the markers are added. The id that is genterated is `sequenceflow-end-white-black`, which is not unique enough to be referenced via `url(#sequenceflow-end-white-black)`. 

I wonder if this id could be generated to include the svg data-element-id to guarantee that the marker is from that SVG?

https://github.com/bpmn-io/bpmn-js/blob/66d2cdd5ddccdd82d79a0f5b5a1c57cde4419ebe/lib/draw/BpmnRenderer.js#L113

**Comment by philippfromme** [comment]
You're right, using multiple instances of bpmn-js on one page leads to problems with the marker IDs. Good catch. We'll look into that.

**Comment by vpanfilov** [comment]
Related to https://github.com/bpmn-io/bpmn-js/issues/639. You can find a CSS workaround there, which I was using in the past. Now I am using my custom renderer, where marker IDs are generated by this line:

```
var id = type + '-' + fill + '-' + stroke + '-' + viewerID;
```
`viewerID` is the unique id of bpmn-js instance.

**Comment by nikku** [comment]
Pretty much addressed by #709.

**Comment by nikku** [comment]
Closed with #709
